### PR TITLE
Create mechanism to automate docker config generation for external registries

### PR DESCRIPTION
**Pull request by m00g3n**
Changes proposed in this pull request:

- Introduces an admission web-hook to create/update `.dockerconfigjson` key in registry configuration secrets.
The value of the will contain docker configuration for kaniko job.

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/10238

### CODE REVIEW

File: resources/serverless/values.yaml [hunk 1]
```diff
@@ -77,8 +77,8 @@ global:
 images:
   manager:
     repository: "eu.gcr.io/kyma-project/function-controller"
-    tag: "PR-10434"
-    pullPolicy: IfNotPresent
+    tag: "PR-10408"
+    pullPolicy: Always
```
Comment (m00g3n) [inline resources/serverless/values.yaml:81]: will be changed in bump

**Review by pPrecel — APPROVED** [review]
LGTM

**Review by pPrecel — APPROVED** [review]

**Pull request merged**